admin-core/src/main/kotlin/com/epam/drill/admin/DrillApplication.kt

```diff
@@ -186,7 +187,7 @@ private fun Application.installPlugins() {
 
 private fun StatusPages.Configuration.defaultStatusPages() {
     exception<Throwable> { cause ->
-        logger.error(cause) { "Build application finished with exception" }
+        logger.error(cause) { "Failed to process the request ${this.context.request.path()}" }
```

Comment:
Can we write more request data to log (`request.path` will only tell us _which_ request handler failed, but not the input parameters - headers, body, cookies, etc)

Reply:
1. Logging body by default is a bad idea. It may contain sensitive data

2. It is best to enable such verbose logging only when necessary - there is a handy ktor lib for that
TODO create task to add ktor-logger
https://github.com/Koriit/ktor-logging